plugin-server/src/main/ingestion-queues/kafka-queue.ts

```diff
@@ -117,13 +117,26 @@ export class KafkaQueue {
                                 "Probably the batch took longer than the session and we couldn't commit the offset"
                             )
                         }
-                        if (
-                            error.message &&
-                            !error.message.includes('The group is rebalancing, so a rejoin is needed') &&
-                            !error.message.includes('Specified group generation id is not valid') &&
-                            !error.message.includes('Could not find person with distinct id')
-                        ) {
-                            Sentry.captureException(error)
+                        if (error.message) {
+                            let logToSentry = true
+                            const messagesToIgnore = [
+                                'The group is rebalancing, so a rejoin is needed',
+                                'Specified group generation id is not valid',
+                                'Could not find person with distinct id',
+                                'The coordinator is not aware of this member',
+                            ]
+                            for (const msg in messagesToIgnore) {
+                                if (error.message.includes(msg)) {
+                                    const metricSuffix = msg.toLowerCase().split(' ').join('_')
```

Comment:
this is not a good idea.
1. we'll get massive metric names
2. one of the messages has a comma

I'd suggest writing a little map of message to suffix:

```js
{
    'The group is rebalancing, so a rejoin is needed': 'group_rebalancing',
    // ...
}
```

Reply:
not exactly, the `msg` comes from the list provided above, so there is no coma (it's not from `error.message`), so they are exactly those 4 from `messagesToIgnore`